TwoPaneLayout/NavigationRail/README.md

```diff
@@ -0,0 +1,71 @@
+# Navigation Rail
+
```

Comment:
It looks like somehow the navigation rail sample was added to the branch as well! I think the whole TwoPaneLayout/NavigationRail folder can be removed?